src/domain/worklets/AudioInputProcessor.ts

```diff
@@ -8,6 +8,12 @@
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 //
 
+import AudioConstants from "../audio/AudioConstants";
+
+/*@devdoc
+ *  see: https://developer.mozilla.org/en-US/docs/Web/API/AudioWorkletGlobalScope
+ */
```

Comment:
```suggestion
// See: https://developer.mozilla.org/en-US/docs/Web/API/AudioWorkletGlobalScope
```

The `/*@devdoc` comments are JSDoc comments that get compiled into the SDK's developer docs using the `npm run devdoc` command.